docs/contributing/index.rst

```diff
@@ -1,7 +1,66 @@
 Contributing
 ============
 
+.. toctree::
+
 Contributing to RestrictedPython
+--------------------------------
+
+Legal requirements to contribute to RestrictedPython
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The projects under the zopefoundation GitHub organization, and therefore RestrictedPython, are open source and welcome contributions in different forms:
+
+* bug reports
+* code improvements and bug fixes
+* documentation improvements
+* pull request reviews
+
+For any changes in the repository besides trivial typo fixes you are required to sign the contributor agreement.
+See https://www.zope.dev/developer/becoming-a-committer.html for details.
+
+Please visit our `Developer Guidelines`_ if you'd like to contribute code changes and our `guidelines for reporting bugs`_ if you want to file a bug report.
+
+
+.. _`Developer Guidelines`: https://www.zope.dev/developer/guidelines.html
+.. _`guidelines for reporting bugs`: https://www.zope.dev/developer/reporting-bugs.html
+
+Preperations for Contributing
++++++++++++++++++++++++++++++
+
+If you want to contribute to this package, please prapare an development entvironment, that includes ``git``, ``tox`` and a posibility to have several Python Versions avaliable, for example ``pyenv``.
+Please read the part of `ref`Understanding How RestrictedPython works internally` first.
+
+Preperations for a new Python Version
++++++++++++++++++++++++++++++++++++++
+
+RestrictedPython should be updated for each new Version of Python.
+To do so, please read the Changelog (`What's new in Python`_) and copy and adjust the new AST Grammar (can be found under: `Python 3 AST`_) to ``/docs/contributing/ast/python<version>.ast`` and add a new ``changes_from<old_version>to<new_version>.rst to /docs/contributing`` and add those to ``index.rst``.
+
+For each new *AST Node* please add a ``visit_<AST Node>`` to ``/src/RestrictedPython/transformer.py`` and add tests to ``/tests/``.
+
+If the new AST Node should be disabled by default, please add a ``visit_<AST Node>`` methode like:
```

Comment:
Is "Node" a formal or proper noun? I think it should be lowercase throughout. I'll let you make the call on that one, as I am not familiar with RP.

```suggestion
If the new AST Node should be disabled by default, add a ``visit_<AST Node>`` method such as the following:
```